CMakeLists.txt

```diff
@@ -0,0 +1,585 @@
+# # CMake project for fdk-aac
+#
+# ## Basic usage
+#
+# 1. Download and install CMake >= 3.10 (CMake >= 3.15 is recommended)
+# 2. Clone fdk-aac repository
+# 3. In fdk-aac directory create and go to `build` subdirectory:
+#
+#    mkdir build
+#    cd build
+#
+# 4. Run cmake to configure project with desired build type (Release|Debug):
+#
+#    cmake .. -DCMAKE_BUILD_TYPE=Release
+#
+#    Note CMake configuration without CMAKE_BUILD_TYPE option will not set build
+#    type to some default and it is most likely is not what you want.
+#
+#    If you want to build static (default is shared library), add BUILD_SHARED_LIBS=OFF option:
+#
+#    #    cmake .. -DBUILD_SHARED_LIBS=OFF
+#
+#    You can combine options as well:
+#
+#    cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF
+#
+# 5. Run cmake to build project:
+#
+#    cmake --build . -c Release
+
+cmake_minimum_required(VERSION 3.10)
+
+# Policies
+
+## Enables CMAKE_MSVC_RUNTIME_LIBRARY option support for CMake >= 3.15
+## if you want to use a MSVC multi-threaded statically-linked runtime library
+## If you enable it, CMake will build fdk-acc.dll without external dependencies.
+##
+## Example usage:
+##
+## cmake .. -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreaded$<$<CONFIG:Debug>:Debug>
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+project(fdk-aac VERSION 2.0.1)
+
+# Includes
+
+include(CheckLibraryExists)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+if(CMAKE_VERSION VERSION_LESS 3.11)
+  set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+  set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+  set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+  include(CPack)
+endif()
```

Comment:
Shouldn't the include be outside of the if/endif?

Reply:
Of course.